### PR TITLE
LPS-102769 Item Selector developer tutorial draft

### PR DESCRIPTION
**Pull request by Alec-Shay**
See: https://issues.liferay.com/browse/LPS-102769

Draft creating a new tutorial for creating an item selector dialog in a sample widget.

I rewrote the code sample (for the JSP at least) to use the new method of creating an item selector (using `Liferay.Util.openSelectionModal`). Please let me know of any more feedback from here. Thanks!

### CODE REVIEW

File: docs/dxp/7.x/en/developing-applications/data-frameworks/implementing-an-item-selector.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+```
+
+This snippet of JavaScript first retrieves the "Select Role" button through its identifier (`portlet:namespace />selectRole`). Then it adds an event listener to create the item selector dialog when clicked.
+
+### Define and Use the Item Selector Dialog
+
+Define the item selector dialog and its behavior inside the select button's on-click function. Use the `Liferay.Util.openSelectionModal` constructor to create the dialog:
+
+```js
+var itemSelectorDialog = new ItemSelectorDialog.default(
```
Comment (ambrinchaudhary) [inline docs/dxp/7.x/en/developing-applications/data-frameworks/implementing-an-item-selector.md:177]: Please, change this line `var itemSelectorDialog = new ItemSelectorDialog.default(` to `Liferay.Util.openSelectionModal`

File: docs/dxp/7.x/en/developing-applications/data-frameworks/implementing-an-item-selector.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+    }
+);
+```
+
+The `onSelect` field must define a function to handle the value when it is clicked. Define the dialog's behavior when the user makes a selection within this function.
+
+The value for the `eventName` field **must** match the String you used with the `RequestBackedPortletURLFactory` in the Java code (in this example, `selectItem`). You must also use the URL that you retrieved from the `request` object previously for the `url` field when defining the item selector dialog.
+
+```tip::
+   If you want your item selector to support selecting multiple items, then you can enable multiple selection by adding this field to the ``ItemSelectorDialog``: ``multiple: true``.
```
Comment (ambrinchaudhary) [inline docs/dxp/7.x/en/developing-applications/data-frameworks/implementing-an-item-selector.md:194]: Change `ItemSelectorDialog` to `openSelectionModal`

**Comment by Alec-Shay** [comment]
Thanks! Pushed up these changes (25ac402) and sending it forward for further review with the KM team.

**Comment by Alec-Shay** [comment]
Forwarded to: https://github.com/sez11a/liferay-learn/pull/201

**Pull request closed without merging**